lib/patch-series-metadata.ts

```diff
@@ -9,7 +9,7 @@ export interface IPatchSeriesMetadata {
     latestTag?: string;
     referencesMessageIds?: string[];
     tipCommitInGitGit?: string;
-    branchNameInGitsterGit?: string;
+    branchNameInMaintainerGit?: string;
```

Comment:
@ttaylorr this would break GitGitGadget: this interface is a description of the data that is stored in [the `refs/notes/gitgitgadget`](https://github.com/gitgitgadget/git/commits/refs/notes/gitgitgadget) notes.

To see an example, take a peek at https://github.com/gitgitgadget/git/blob/8e58afdf46074debd5cf65693f2906d6dc67cf02/4c/c9/8ffa1e29134751b57d34fc8dd723bf7db1cc

If you rename the field in this interface (and in all of its users in GitGitGadget's code base), every time GitGitGadget reads the notes it will assume that that information is missing. That's a bug.